### ISSUE TITLE
`filestack_js_init_tag` creates incorrect JS code when including security policy

### ISSUE THREAD

**Issue by diegous** [filestack/filestack-rails#205]
`filestack_js_init_tag` generates JS code like this one
```js
var filestack_client = filestack.init('someApiKey', {'signature': 'xxx', 'policy': 'yyy'}, '');
```
but that is not the format that the js client accepts, the signature and policy object should be inside another object referenced bye the `security` key, like this
```js
var filestack_client = filestack.init('someApiKey', { 'security': {'signature': 'xxx', 'policy': 'yyy'}}, '');
```
as shown in [this doc page](https://www.filestack.com/docs/tutorials/security-best-practices/)

**Comment by diegous** [comment]
Added PR with fix for this #206

**Comment by gabifiolek** [comment]
@diegous - it is already fixed in version 4.0.4.
Thank you for your PR!
